CMake/AbseilFindDependencies.cmake

```diff
@@ -0,0 +1,121 @@
+            BUILD_COMMAND "make" "PREFIX=${CCTZ_BUNDLE_INTERNAL_INSTALLDIR}"
+                "SRC=${CCTZ_BUNDLE_INTERNAL_SRCDIR}/" "-f" "${CCTZ_BUNDLE_INTERNAL_SRCDIR}/Makefile"
+                "CXX=${CMAKE_CXX_COMPILER}" "CC=${CMAKE_C_COMPILER}"
+            INSTALL_DIR "${CCTZ_BUNDLE_INTERNAL_INSTALLDIR}"
+            INSTALL_COMMAND "make" "install"
+                "PREFIX=${CCTZ_BUNDLE_INTERNAL_INSTALLDIR}" "SRC=${CCTZ_BUNDLE_INTERNAL_SRCDIR}/" "-f" "${CCTZ_BUNDLE_INTERNAL_SRCDIR}/Makefile"
+                "CXX=${CMAKE_CXX_COMPILER}" "CC=${CMAKE_C_COMPILER}"
+         )
+
+
+        list(APPEND ABSEIL_PRE_DEPENDENCIES CCTZ_BUNDLE)
+
+        set(CCTZ_INCLUDE_DIRS "${CCTZ_BUNDLE_INTERNAL_INSTALLDIR}/include")
+        set(CCTZ_LIBRARIES "${CCTZ_BUNDLE_INTERNAL_INSTALLDIR}/lib/libcctz.a")
+
+        install(FILES "${CCTZ_BUNDLE_INTERNAL_INSTALLDIR}/lib/libcctz.a" DESTINATION ${CMAKE_INSTALL_FULL_LIBDIR})
+        install(DIRECTORY "${CCTZ_BUNDLE_INTERNAL_INSTALLDIR}/include/cctz" DESTINATION ${CMAKE_INSTALL_FULL_INCLUDEDIR})
+    endif(NOT CCTZ_FOUND)
+endif(NOT ABSL_ENABLE_BUNDLE)
+
+include_directories(${CCTZ_INCLUDE_DIRS})
+
+
+
+if(BUILD_TESTING)
+
+
+    if(NOT ABSL_ENABLE_BUNDLE)
+
+        find_package(GTest REQUIRED)
+        find_package(GMock REQUIRED)
+
+    else(NOT ABSL_ENABLE_BUNDLE)
+        # check for GTest
+        find_package(GTest QUIET)
+        find_package(GMock QUIET)
+
+        # embedded dep
+        if((NOT GTEST_FOUND) OR (NOT GMOCK_FOUND))
+            # if not found, try to bundle it
+            message(STATUS "No External GTest found, Use bundle GTest package")
+
+            set(GTEST_BUNDLE_INTERNAL_INSTALLDIR "${CMAKE_CURRENT_BINARY_DIR}/GTEST_BUNDLE-prefix/install")
+
+            ExternalProject_Add(GTEST_BUNDLE
+                GIT_REPOSITORY "https://github.com/google/googletest/"
+                GIT_TAG "master"
+                CMAKE_ARGS "-DCMAKE_INSTALL_PREFIX=${GTEST_BUNDLE_INTERNAL_INSTALLDIR}"
+                INSTALL_DIR "${GTEST_BUNDLE_INTERNAL_INSTALLDIR}"
+                LOG_DOWNLOAD 1
+                LOG_BUILD 1
+             )
+
+             list(APPEND ABSEIL_PRE_DEPENDENCIES GTEST_BUNDLE)
+
+             set(GTEST_INCLUDE_DIRS "${GTEST_BUNDLE_INTERNAL_INSTALLDIR}/include")
+             set(GMOCK_INCLUDE_DIRS "${GTEST_BUNDLE_INTERNAL_INSTALLDIR}/include")
+
+             set(GTEST_LIBRARIES "${GTEST_BUNDLE_INTERNAL_INSTALLDIR}/lib/libgtest.a")
+             set(GTEST_MAIN_LIBRARIES "${GTEST_BUNDLE_INTERNAL_INSTALLDIR}/lib/libgtest_main.a")
+             set(GMOCK_LIBRARIES "${GTEST_BUNDLE_INTERNAL_INSTALLDIR}/lib/libgmock.a")
+
+        endif((NOT GTEST_FOUND) OR (NOT GMOCK_FOUND))
+
+    endif(NOT ABSL_ENABLE_BUNDLE)
+
+    include_directories(${GTEST_INCLUDE_DIRS} ${GMOCK_INCLUDE_DIRS})
+
+endif()
+
+# dummy target, use to give at least one target to ABSEIL_PRE_DEPENDENCIES
+add_custom_target(DUMMY_TARGET_BUNDLE COMMAND "true")
```

Comment:
Using `true` looks like the kind of things that will just complicate porting to Windows.

Reply:
Removed.